pkg/interface/src/views/components/AudioPlayer.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { Action, Text, Icon, Row } from '@tlon/indigo-react';
+
+function formatTime(num: number) {
+  const minutes = Math.floor(num / 60);
+  const seconds = Math.floor(num % 60);
+  return `${minutes}:${seconds.toString().padStart(2, '0')}`;
+}
+
+export function AudioPlayer(props: { url: string; title?: string }) {
```

Comment:
Are there any controls in the top bar or does audio only play until you navigate away from the collection?

Reply:
No, playback is scoped to component. Will refactor at some point to have a global context for audio playback